src/components/Info/Info.tsx

```diff
@@ -0,0 +1,41 @@
+import { motion } from 'framer-motion';
+import React from 'react';
+import { info } from '../../data/info'; // Importing the info data
+
+interface InfoContentProps {
+  children?: React.ReactNode;
+}
+
+const InfoContent: React.FC<InfoContentProps> = () => {
+  return (
+    <div
+      className="flex flex-col"
+      style={{ textShadow: '#000000 2px 2px 0px, #262525 2px 2px 0px' }}>
```

Comment:
Make this a custom tailwind style

Reply:
tailwindcss doesnt support text shadow, this was the easy (or lazy) way